django/test/runner.py

```diff
@@ -696,5 +697,7 @@ def filter_tests_by_tags(suite, tags, exclude_tags):
             matched_tags = all_tags.intersection(tags)
             if (matched_tags or not tags) and not all_tags.intersection(exclude_tags):
                 filtered_suite.addTest(test)
+                if verbosity >= 2:
```

Comment:
Why not have this happen for default verbosity?

Looks look a good change otherwise!